grpcproxy/grpcproxy_test.go

```diff
@@ -0,0 +1,62 @@
+// +build !race
+
+package grpcproxy
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	gr "runtime"
+	"testing"
+
+	"strings"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/satori/go.uuid"
+)
+
+var buildScript string = `#!/bin/bash
+cd ../plugin_build
+go build --tags="dummy" --buildmode=plugin -o $1.so
+`
```

Comment:
Wait, a two-line script? This would be trivial to do in a `exec.Cmd`. You can specify a different dir with `cmd.Dir` and the args can be passed as you'd expect.